cmd/signature-v4-parser.go

```diff
@@ -46,6 +47,32 @@ func (c credentialHeader) getScope() string {
 	}, "/")
 }
 
+func getReqAccessKey(r *http.Request, region string) (string, APIErrorCode) {
```

Comment:
should be fixed to work with V2 requests too

Reply:
done